vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -775,6 +832,27 @@ static void StyleYogaNode(
       float result = NumberOrDefault(value, 0.0f /*default*/);
 
       YGNodeStyleSetBorder(yogaNode, YGEdgeBottom, result);
+    } else if (key == "experimental_layoutConformance") {
+      // Reset the current layout conformance value to prepare for visitors.
+      shadowNode.m_layoutConformance = std::nullopt;
+      auto layoutConformance = value.AsString() == "strict";
```

Comment:
What does string coercion do for null value?

Reply:
Empty string.